assets/queries/cloudFormation/aws/cloudfront_viewer_protocol_policy_allows_http/metadata.json

```diff
@@ -1,9 +1,9 @@
 {
   "id": "31733ee2-fef0-4e87-9778-65da22a8ecf1",
-  "queryName": "Viewer Protocol Policy Allows HTTP",
+  "queryName": "Cloudfront Viewer Protocol Policy Allows HTTP",
```

Comment:
Hello @jycamier,

Thank you so much for continuing to contribute 🚀 

In order to pass the tests, you will need to update the `queryName` in the `positive_expected_result.json`

Reply:
sure ! :)